File: find_average_in_array.py
list1 = [1,2,3,4]

def find_average(lst):
	sum = float(0)
	for i in lst:
		sum = sum + i

	return sum/len(lst)

print find_average(list1)


# Challenge: Given as array of integers, write a function that finds the average of all members.

File: find_average_in_array.js
arr = [1,2,3,4]
		
function findAverage() {
	var sum = 0;
	for (var i = 0; i < arr.length; i++) {
		sum = sum + arr[i]
	}
	return sum/arr.length;
}

console.log(findAverage(arr));

// Challenge: Given as array of integers, write a function that finds the average of all members.